src/Tizen.Multimedia.MediaPlayer/Player/AudioEffect.cs

```diff
@@ -26,36 +26,23 @@ namespace Tizen.Multimedia
     public class AudioEffect
     {
         private readonly EqualizerBand[] _bands;
+        private readonly int _count;
+        private readonly bool _isAvailable;
+        private readonly Range _bandLevelRange;
```

Comment:
_count, _isAvailable, _bandLevelRange

Why need it as filed member?
nobody use it, except constructor
I think, it should be a local variable

Reply:
that's right. i'll fix it.